src/excmds.ts

```diff
@@ -3236,6 +3236,7 @@ export function comclear(name: string) {
 */
 //#background
 export function bind(...args: string[]) {
+    clear_keymap_cache()
```

Comment:
Bad idea IMO. I'm pretty sure we're eventually going to be adding/removing bindings from other commands (actually we already do, with bindurl/unbindurl). I think this should be done in a changeListener instead (or at least in config.set).